var/spack/repos/builtin/packages/hdf/package.py

```diff
@@ -66,4 +65,5 @@ def install(self, spec, prefix):
         configure(*config_args)
 
         make()
-        make("install")
+        make('check')
```

Comment:
I think as we add more platforms like BG/Q and Xeon Phi, it's going to be harder to just run `make check`, as `make check` typically assumes the generated binaries can actually run on the build host.  I would recommend wrapping these by default.